openupgradelib/openupgrade.py

```diff
@@ -2680,18 +2680,27 @@ def delete_record_translations(cr, module, xml_ids, field_list=None):
                     list_columns.pop(i)
             if not list_columns:
                 continue
+            # In case list_columns only have 1 column
+            # we need to remove the parenthesis to avoid error
+            if len(list_columns) > 1:
+                columns = ", ".join(list_columns)
+                values = ", ".join(
+                    "jsonb_build_object('en_US', {x} -> 'en_US')".format(x=x)
+                    for x in list_columns
+                )
+                columns = "({})".format(columns)
+                values = "({})".format(values)
+            else:
+                columns = list_columns[0]
+                values = "jsonb_build_object('en_US', {x} -> 'en_US')".format(
+                    x=list_columns[0]
+                )
```

Comment:
```suggestion
            columns = ", ".join(list_columns)
            values = ", ".join(
                "jsonb_build_object('en_US', {x} -> 'en_US')".format(x=x)
                for x in list_columns
            )
            if len(list_columns) > 1:
                columns = "({})".format(columns)
                values = "({})".format(values)
```

Reply:
Thank you, but i prefer keeping my comment is that ok?

Reply:
Got it, done sir